import-export/exercise/Car.js

```diff
@@ -0,0 +1,20 @@
+import Vehicle from './Vehicle.js';
+
+class Car extends Vehicle {
+    constructor(brand, speed = 0, fuelType,) {
+        super(brand, speed);
+        this.fuelType = fuelType;
+    }
+
+    signal() {
+        return `Beep beep! This is a ${this.brand} car.`
+    }
+
+    showInfo() {
+        return `${this.brand} has speed ${this.speed} and use ${this.fuelType}`;
+    }
+}
+
+// In file where class Car is implemented export new instance of class with already defined values
+// (Not class should be imported, instance of class)
+export default new Car('Peugeot', 20, 'Petrol');
```

Comment:
Your solution is fine but in future use named export instead of default export: 
export const car = new Car('Peugeot', 20, 'Petrol');

There are several cases why it is better: 
1. You define name for exported instance of class and other person use defined name instead of creating personal one (example: newCar, testCar, etc..) 
2. IDE autocomplete helps you easier import file 
3. Possibility import several files using index.ts file behevior (Will explain it in 2nd api lecture)